pulp_2to3_migration/app/json_schema.py

```diff
@@ -51,9 +51,9 @@
                                     }
                                 }
                             },
-                            "required": ["name", "repository_versions"],
+                            "required": ["name", "importer_repository_id", "repository_versions"],
                             "additionalProperties": false,
-                            "$comment": "pulp2_repository_id field should be specified so we know which importer to use when migrating multiple pulp2repos into repo versions",
+                            "$comment": "importer_repository_id field should be specified so we know which importer to use when migrating multiple pulp2repos into repo versions",
                             "if": {
                             "properties": {
```

Comment:
Does line 67 need to be changed?  it looks like it's referring to the nested one.  But if that's the case then we might be able to get rid of the conditional because it's already required on line 45?

Reply:
yeah, I think all conditional can go away. @ipanova , should confirm :)

Reply:
i confirm ^_^

Reply:
you can remove if/then conditionals